Kickstarter-iOS/Library/QualtricsTypes.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import Library
+import Qualtrics
+
+public struct QualtricsConfigData: Equatable {
+  public let brandId: String
+  public let zoneId: String
+  public let interceptId: String
+  public let stringProperties: [String: String]
+}
+
+enum QualtricsIntercept {
+  case survey
+
+  var interceptId: String {
+    switch AppEnvironment.current.environmentType {
+    case .production:
+      return "SI_eUKtARDK3785TQF"
```

Comment:
These intercept IDs are not sensitive because they can easily be observed on the web by inspecting traffic between the JS SDK and Qualtrics API.

Reply:
Also - this is the one value that we expect to change which is also why it isn't in Secrets. Later we may even obtain this dynamically and configure our surveys a bit differently. It seems a little strange to me that the `interceptId` forms part of the configuration data for the singleton but 🤷‍♂ that appears to be how it works.